src/main/java/com/kevin/tiertagger/TierTagger.java

```diff
@@ -15,12 +18,15 @@
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 public class TierTagger implements ModInitializer {
     public static final URI ENDPOINT = URI.create("https://api.uku3lig.net/tiers/vanilla");
-
+    public static final MinecraftClient MC = MinecraftClient.getInstance();
+    public static final LinkedHashMap<GameProfile, SignedMessage> messagesWithAuthor = new LinkedHashMap<>();
```

Comment:
you don't need a `LinkedHashMap`, a simple `HashMap` could work just fine, since you don't need to keep everything in order

Reply:
Yes I do, since it might remove the visible messages, when it removes messages once it reachs 100+ messages